src/Vts/MonteCarlo/Tissues/SingleInclusionTissue.cs

```diff
@@ -63,21 +63,17 @@ public override int GetRegionIndex(Position position)
         public override int GetNeighborRegionIndex(Photon photon)
         {
             // first, check what region the photon is in
-            int regionIndex = photon.CurrentRegionIndex;
+            var regionIndex = photon.CurrentRegionIndex;
 
             // if we're in the layer region of the inclusion, could be on boundary of layer
             if ((regionIndex == _layerRegionIndexOfInclusion) && !Regions[_layerRegionIndexOfInclusion].OnBoundary(photon.DP.Position) )
             {
                 return _inclusionRegionIndex;
             }

```

Comment:
This if is really long so this one might look better with brackets, thoughts?

Reply:
I kept brackets and used wrap line for "if".